DeploymentSteps.md

```diff
@@ -12,7 +12,8 @@ Don't delete old sections - just leave them towards the bottom of the file, and
 
 ## Next (pending changes for the next coming release)
 
-1. Keycloak needs to update realm display name to UID2 Portal
+1. Admin must be deployed before the portal to include the Role changes needed for setting client types (sas-UID2-1782-approved-types-to-admin)
+2. Keycloak needs to update realm display name to UID2 Portal
```

Comment:
I forgot about this file - I have since updated the realm display name on integ & prod so you can move this line to a new section below (I guess with the heading 0.7.5 which is the current version?). Or I can update it in a separate PR.